README.md

```diff
@@ -37,28 +37,29 @@ git clone --recursive https://github.com/clara-genomics/AtacWorks.git
 
 #### Install dependencies
 
-* Download `bedGraphToBigWig` and `bigWigToBedGraph` binaries and add to your $PATH
+* Download `bedGraphToBigWig` and `bigWigToBedGraph` binaries and add to your $PATH to your bashrc.
```

Comment:
Confused about this sentence. Is there an 'and' or 'or' missing?

Reply:
Changed it to `and add $PATH to your bashrc`